src/components/IntersectionObserver.vue

```diff
@@ -0,0 +1,51 @@
+<!--
+	SPDX-FileCopyrightText: 2023 Ferdinand Thiessen <rpm@fthiessen.de>
+	SPDX-License-Identifier: AGPL-3.0-or-later
+-->
```

Comment:
Curious if there's a format for this!?

Reply:
My IDE is generating this on new files based on the https://reuse.software/ conventions.
(But I also think reuse is for example used for the app template you can download from the [Nextcloud appstore](https://apps.nextcloud.com/developer/apps/generate)).